history/history-network.md

```diff
@@ -183,8 +182,21 @@ HistoricalRootsBlockProof = Container[
     slot: Slot
 ]
 
-BlockHeaderProof = Union[None, HistoricalHashesAccumulatorProof, HistoricalRootsBlockProof]
+# Proof that BeaconBlock root is part of HistoricalRoots and thus canonical
+# For Capella and onwards
+HistoricalSummariesProof = Vector[Bytes32, 13]
```

Comment:
What would you think about renaming this to something like `HistoricalSummariesRootsProof` or something similar so it's just 100% clear what it is?  In parallel, I wonder if we could rename this [beacon chain object](https://github.com/ethereum/portal-network-specs/pull/291/files) to `HistoricalSummariesWithStateProof` to be 100% clear this is the proof that the summaries are rooted in the current beacon state.

Reply:
In Trin we use `HistoricalSummariesBlockProof` and `HistoricalSummariesStateProof`. Happy to switch to whatever we decide.

Reply:
I think the `HistoricalSummariesBlockProof` is the container below that holds both the `BeaconBlockProof` (i.e. the proof that an EL blockhash is part of a beacon block and then what is currently called thethe `HistoricalSummariesProof` which anchors the Beacon Block root in the `HistoricalSummaries`.  As such, was thinking maybe we can call this second one "HistoricalSummariesRootProof" to reflect that we're anchoring the beacon block root in the particular `block_summary_root` for the period in which it falls.

Reply:
I've altered the naming which should avoid the name clash and hopefully also be more clear.